src/Microsoft.CSharp/pkg/Microsoft.CSharp.pkgproj

```diff
@@ -31,6 +33,13 @@
     <InboxOnTargetFramework Include="xamarinwatchos10">
       <AsFrameworkReference>true</AsFrameworkReference>
     </InboxOnTargetFramework>
+    <HarvestIncludePaths Include="ref/netcore50;lib/netcore50" />
+    <HarvestIncludePaths Include="ref/netstandard1.0" />
+    <HarvestIncludePaths Include="lib/netstandard1.3" />
+    <ValidatePackageSuppression Include="PermitMissingImplementation">
+      <!-- Supported inbox by all ns1.0-1.2 frameworks -->
```

Comment:
This case is rare enough that I wanted to make it a suppression.  Ideally we'd actually have a matching impl and let inbox override that.